lnwallet/channel.go

```diff
@@ -1576,12 +1592,18 @@ func (lc *LightningChannel) AddHTLC(htlc *lnwire.HTLCAddRequest) (uint32, error)
 		return 0, err
 	}
 
+	var isDust bool
+	if htlc.Amount < lc.channelState.DustLimit {
```

Comment:
As far as I understand we negotiate the dust limit on funding request/response messages. After that we have identical dust limit on both sides and may make the decision - should we include HTLC in commitment transaction or not - relaying on this negotiated dust limit, could you please elaborate why we need have different dust limit for both sides?